scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1319,6 +1300,8 @@ Else
     $BypassInstallationFlag = $null
 }
 
+TestSetup
```

Comment:
This may break if test name is "All".  Please revert this change.

Reply:
Got it. Reverted the change.